src/main/java/com/auth0/client/mgmt/ResourceServerEntity.java

```diff
@@ -0,0 +1,137 @@
+package com.auth0.client.mgmt;
+
+import java.util.List;
+
+import com.auth0.json.mgmt.ResourceServer;
+import com.auth0.net.CustomRequest;
+import com.auth0.net.Request;
+import com.auth0.net.VoidRequest;
+import com.auth0.utils.Asserts;
+import com.fasterxml.jackson.core.type.TypeReference;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+
+public class ResourceServerEntity  {
+    private OkHttpClient client;
+    private HttpUrl baseUrl;
+    private String apiToken;
+
+    ResourceServerEntity(OkHttpClient client, HttpUrl baseUrl, String apiToken) {
+        this.client = client;
+        this.baseUrl = baseUrl;
+        this.apiToken = apiToken;
+    }
+
+    /**
+     * Creates request to fetch all resource servers.
+     * See <a href=https://auth0.com/docs/api/management/v2#!/Resource_Servers/get_resource_servers>API documentation</a>
+     *
+     * @return request to execute
+     */
+    public Request<List<ResourceServer>> list() {
+
+        HttpUrl.Builder builder = baseUrl
+                .newBuilder()
+                .addPathSegments("api/v2/resource-servers");
+
+        String url = builder.build().toString();
+        CustomRequest<List<ResourceServer>> request = new CustomRequest<>(client, url, "GET",
+                                                                          new TypeReference<List<ResourceServer>>() {
+                                                                          });
+        request.addHeader("Authorization", "Bearer " + apiToken);
+        return request;
+    }
+
+    /**
+     * Cretes request for fetching single resource server by it's ID.
+     * See <a href=https://auth0.com/docs/api/management/v2#!/Resource_Servers/get_resource_servers_by_id>API documentation</a>
+     *
+     * @param resourceServerId {@link ResourceServer#id} field
+     * @return request to execute
+     */
+    public Request<ResourceServer> get(String resourceServerId) {
```

Comment:
**All** these entity methods that have a `resourceServerId` as a parameter, I'm checking the API docs and it says that it could be either the id or the identifier (audience) value. Assert this is true and in that case, rename to `resourceServerIdOrIdentifier` and explain this in the javadoc.

Reply:
done